index.html

```diff
@@ -0,0 +1,155 @@
+am aut amet ant fugiatum, utem non reptat ad magnienecto volorent fugia nonsedi gendand itatquos as magnimpor aut molor autatia ad est officimus accae et omnia cus, od qui ate
+</p>
+
+          </article>
+
+          <article class="div2">
+
+            <img class="dogimg" click="info(name = 'Murphy', breed = 'Boxer', price = '$123.45')" src="./images/murphy-card.jpg" alt="dog running">
+
+            <h3>Murphy</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" click="dogprice(price = '$123.45')" class="Adopt-link">Adopt</a>
+          </article>
+
+          <article class="div2">
+
+            <img class="dogimages" src="./images/poppy-card.jpg" alt="dog sitting">
+
+            <h3>Poppy</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link">Adopt</a>
+          </article>
+
+          <article class="div2">
+
+            <img class="dogimages"  src="./images/jack-card.jpg" alt="he is staring at the camera">
+
+            <h3>Jack</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link" >Adopt</a>
+          </article>
+
+          <article>
+
+            <img class="dogimages" src="./images/duffy-card.jpg" alt="Black dog is staring at the camera" >
+
+            <h3>Duffy</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link">Adopt</a>
+          </article>
+
+          <article class="div2">
+
+            <img class="dogimages" src="./images/lucas-card.jpg" alt="white dog is looking away" >
+
+            <h3>Lucas</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link">Adopt</a>
+          </article>
+
+          <article class="div2">
+
+            <img class="dogimages" src="./images/jake-card.jpg" alt="pitbull is standing">
+
+            <h3>Jake</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link">Adopt</a>
+          </article>
+
+          <article class="div2">
+
+            <img class="dogimages" src="./images/angus-card.jpg" alt="dog with black bakcground">
```

Comment:
You'll want to use some width percentage on these images so they all fit the boxes right.